components/RichTextEditor.js

```diff
@@ -291,6 +300,16 @@ export default class RichTextEditor extends React.Component {
     return e;
   };
 
+  handleLink = () => {
+    const urlInput = document.querySelector("[data-trix-input][name='href']");
+    const urlInputValue = urlInput.value;
+
+    // Automatically add 'https://' to the url
+    if (!urlInputValue.startsWith('http://') && !urlInputValue.startsWith('https://')) {
```

Comment:
@Jeanluis019 : I think you can simplify this to, 

```suggestion
    if (!urlInputValue.startsWith('http')) {
```

Reply:
Improvement applied. Thank you!